src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,56 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's contact in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidContact(String, String)}
+ */
+public class Contact {
+
+    public final String value;
```

Comment:
Try to make it private. 
In general, start off with private members, then use protected or public only if absolutely necessary.